ferrocene/ci/split-tasks.py

```diff
@@ -210,7 +210,9 @@ def cli():
     else:
         flags += get_flags_for_default_job(JOBS_DEFINITION, split[0])
 
-    print(" ".join(flags))
+    flag_arg = " ".join(flags)
+    print(f"Split tasks: {flag_arg}", file=sys.stderr)
+    print(flag_arg)
```

Comment:
did you mean to repeat displaying `flag_arg` here

Reply:
Yes, one will output to stdout (which is being bash expanded) and the other will get printed to console. So folks can see what is being run. THat was the request. :)